examples/python-ros2-dataflow/control_node.py

```diff
@@ -18,10 +18,14 @@
                     f"""Pose: {event["value"]}""".replace("\r", "").replace("\n", " ")
                 )
             case "tick":
+                direction = {
+                    "linear": {
+                        "x": 1.0,
+                    },
+                    "angular": {"z": 1.0},
+                }
```

Comment:
Is it intentional that the movement is no longer randomized?

Reply:
It's a dev artifact sorry